setup.cfg

```diff
@@ -8,7 +8,7 @@ ignore =
 
 [metadata]
 requires-dist =
-    clint
+    tqdm
```

Comment:
Can you have this match the version in `install_requires`. I'd really rather not use newer features of very recent versions of setuptools, e.g., `~=`. Can we just use `>=`, please?